bigquery/biglake/bigquery_create_partitioned_biglake_table/main.tf

```diff
@@ -90,6 +90,11 @@ resource "google_bigquery_dataset" "default" {
 # This creates a BigQuery table named "my_table" in the dataset "default".
 # The table has a single column named "column1", which is of type STRING
 # and is nullable.
+# The table has Automatic Metadata Caching enabled.
+# https://cloud.google.com/bigquery/docs/biglake-intro#metadata_caching_for_performance
+# `metadata_cache_mode` may be set to `MANUAL` and `max_staleness` ommitted to
```

Comment:
"omitted".

Also may be clearer if we said "Set `metadata_cache_mode` to `MANUAL` and omit `max_staleness` to use manual metadata refresh. Omit both `metadata_cache_mode` and `max_staleness` to disable metadata caching." here and in similar other descriptions below. But I'll defer the exact wording to @adhiggs.

Reply:
Just a suggestion. Use comments to explain details that are not so obviation or details that are not found in reference docs.

Usually users refer to [docs in Hashicorp](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/bigquery_table). So, avoid explaining sample details as in Hashicorp

Reply:
Agree with msampathkumar; we can omit this sort of implementation detail. The important thing is that the code should run as little (preferably none) customization needed from the user.

Reply:
done.
simplified